services/spar/src/Spar/Scim/User.hs

```diff
@@ -610,6 +605,65 @@ assertHandleNotUsedElsewhere hndl uid = do
   unless ((userHandle =<< musr) == Just hndl) $
     assertHandleUnused' "userName does not match UserId" hndl uid
 
+-- | Helper function that given a brig user, creates a scim user on the fly or returns
```

Comment:
well; it always creates a scim user on the fly now right? as we dont store them anymore. Comment should reflect that new reality

Reply:
still valid